tests/system/HTTP/CURLRequestTest.php

```diff
@@ -1156,6 +1156,18 @@ public function testHTTPv2(): void
         $this->assertSame(CURL_HTTP_VERSION_2_0, $options[CURLOPT_HTTP_VERSION]);
     }
 
+    public function testHTTPv3(): void
+    {
+        $this->request->request('POST', '/post', [
+            'version' => 3.0,
+        ]);
+
+        $options = $this->request->curl_options;
+
+        $this->assertArrayHasKey(CURLOPT_HTTP_VERSION, $options);
+        $this->assertSame(CURL_HTTP_VERSION_3, $options[CURLOPT_HTTP_VERSION]);
```

Comment:
Suppress the errors by Psalm/PHPStan with the annotations. 
Because the constant is dynamically defined in PHP 8.1 to 8.3.